### PR TITLE
updated README for v0.18.0

### LINKED ISSUE #677: Documentation: Update confusing information regarding "custom-elements-es5-adapter.js"
In polymer-cli, it is mentioned that `custom-elements-es5-adapter.js` is used for older browsers, while in the webcomponentsjs documentation it is clearly stated, that its used for ES6 (ES2015+) supporting browsers to let class inheretence work properly on these native implementations.

polymer-cli readme - has a mistake concerning `custom-elements-es5-adapter.js` usage*.
https://github.com/Polymer/polymer-cli#--js-compile
```--js-compile
When your application uses the --js-compile CLI flag (or polymer.json config property) your application is compiled from ES6 to ES5. For this to work properly, some older browsers need the custom-elements-es5-adapter.js shim in addition to the webcomponents-loader.js/webcomponents-lite.js polyfill. The build process will automatically inject this ES5 adapter into your application when --js-compile is enabled so that your application works as expected on older browsers.
```

webcomponentsjs: 
https://github.com/webcomponents/webcomponentsjs/tree/v1.0.0-rc.8#custom-elements-es5-adapterjs

### PR DESCRIPTION
**Pull request by FredKSchott**
Resolves #677

README was very out of date, both in terms of CLI behavior and missing references to our new polymer-project.org documentation. I wrote the new README to be more focus and hopefully need less maintenance going forward.

The diff isn't super helpful since it's a rewrite, so here's a link to the full document: https://github.com/Polymer/polymer-cli/blob/ce1255bff15f7e076d364109b918b2cca2710cf5/README.md

Trying to get this merged today if possible for a `latest` release tomorrow, so please review!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,79 +3,49 @@
...
-
-### App-shell Structure
-
-App-shell apps are currently the preferred style for Polymer CLI, and most commands are being optimized to support them. App-shell apps usually have client-side routing (see the [app-route](https://github.com/PolymerElements/app-route) element), and lazy load parts of the UI on demand.
-
-Polymer-CLI supports this style by understand these different types of files:
+```bash
+$ npm install -g polymer-cli
+# or...
+$ yarn add global polymer-cli
```
Comment (rictic) [inline README.md]: Should we put yarn first, as it's such a significant installation performance win?

File: README.md [hunk 2]
```diff
@@ -3,79 +3,49 @@
...
-A command-line tool for Polymer projects.
+The command-line tool for Polymer projects and Web Components.
 
-For detailed help, check out the
-[Polymer CLI](https://www.polymer-project.org/1.0/docs/tools/polymer-cli)
-guide.
+## Features
 
-## Overview
+  - **init** - Initialize a Polymer project from custom templates
```
Comment (aomarks) [inline README.md]: Maybe "from pre-configured starter templates" or something? "custom" might be a little unclear.

File: README.md [hunk 3]
```diff
@@ -3,79 +3,49 @@
...
+  - **install** - Install bower dependencies as well as [dependency variants](https://www.polymer-project.org/2.0/docs/glossary#dependency-variants) for testing
+  - **serve**	- Serve elements and applications locally
+  - **lint** - Lint a project to catch errors before your users
```
Comment (aomarks) [inline README.md]: Nit: "before your users do"

File: README.md [hunk 4]
```diff
@@ -3,79 +3,49 @@
...
 
-For detailed help, check out the
-[Polymer CLI](https://www.polymer-project.org/1.0/docs/tools/polymer-cli)
-guide.
+## Features
 
-## Overview
+  - **init** - Initialize a Polymer project from custom templates
+  - **install** - Install bower dependencies as well as [dependency variants](https://www.polymer-project.org/2.0/docs/glossary#dependency-variants) for testing
+  - **serve**	- Serve elements and applications locally
```
Comment (rictic) [inline README.md]: s/locally/for development/?

File: README.md [hunk 5]
```diff
@@ -89,203 +59,101 @@ You can specify the project files in `polymer.json` so that commands like `polym
...
-Initializes a Polymer project from one of several templates.
 
-Choose a template from a menu:
+### `polymer install [--variants]`
 
-    $ polymer init
+Install your dependencies, similar to running `bower install`.
 
-Create a new project from the 'element' template:
+If the `--variants` option is provided, the command will also search your project's `bower.json` for  a `"variants"` property and install any [dependency variants](https://www.polymer-project.org/2.0/docs/glossary#dependency-variants) listed there as well as well. This is useful if you need to test your elements against multiple versions of Polymer and/or sets of dependencies.
```
Comment (aomarks) [inline README.md]: nit: extra space between "for" and "a"

File: README.md (hunk 3 above)
Comment (rictic) [inline README.md]: How about

> Lint a project to find and diagnose errors quickly

File: README.md [hunk 6]
```diff
@@ -3,79 +3,49 @@
...
-  * shell - The actual app shell, which includes the top-level logic, routing, and so on.
-  * fragments - lazy loaded parts of the application, typically views and other elements loaded on-demand.
 
 ## Configuration
 
-The project files are specified either as global flags: `--entrypoint`, `--shell` and zero or more `--fragment` flags, or in a `polymer.json` configuration file.
+An application can be configured for all commands via global flags: `--entrypoint`, `--shell`, etc.
 
-### polymer.json
+We recommend saving your configuration to a `polymer.json` configuration file in your project so that they can be read automatically for every command. Other project settings, like build and lint rules, can also be defined here. Read the [polymer.json spec](https://www.polymer-project.org/2.0/docs/tools/polymer-json) for a full list of all supported configurations.
```
Comment (rictic) [inline README.md]: of all supported configurations -> of all supported fields

`configuration` sounds modal but the linked doc is closer to a listing of fields, what they mean and how to use them.

File: README.md (hunk 2 above)
Comment (FredKSchott) [inline README.md]: +1, I like that much better

File: README.md [hunk 7]
```diff
@@ -89,203 +59,101 @@ You can specify the project files in `polymer.json` so that commands like `polym
...
+### `polymer help [COMMAND]`
 
-`polymer test` will run the test suite once for each variant, ensuring that you
-have test coverage against supported configurations.
+Run `polymer help` to get a helpful list of supported commands. Pass it a command name (ex: `polymer help serve`) to get detailed information about that command and the options it supports.
 
-## Commands
 
-### help
+### `polymer init [TEMPLATE]`
```
Comment (rictic) [inline README.md]: recommend just `polymer init`, as that's the right way to use it for most people isn't it? the menu is way friendlier than needing to know up front the template name

File: README.md [hunk 8]
```diff
@@ -89,203 +59,101 @@ You can specify the project files in `polymer.json` so that commands like `polym
...
-    $ polymer init
+Install your dependencies, similar to running `bower install`.
 
-Create a new project from the 'element' template:
+If the `--variants` option is provided, the command will also search your project's `bower.json` for  a `"variants"` property and install any [dependency variants](https://www.polymer-project.org/2.0/docs/glossary#dependency-variants) listed there as well as well. This is useful if you need to test your elements against multiple versions of Polymer and/or sets of dependencies.
 
-    $ polymer init element
 
-You can download and run templates built by our community as well. [Search npm](https://www.npmjs.com/search?q=generator-polymer-init) for the full list.
+### `polymer serve [options...]`
```
Comment (rictic) [inline README.md]: [options...] is implicit in all of these yeah?

File: README.md [hunk 9]
```diff
@@ -89,203 +59,101 @@ You can specify the project files in `polymer.json` so that commands like `polym
...
 
-### install
+Start a development server designed for serving Polymer & Web Component projects. Applications are served as-is, while elements are served from a special route where it can properly reference its dependencies.
```
Comment (rictic) [inline README.md]: We should also mention that it does automatic compilation by default.

File: README.md [hunk 10]
```diff
@@ -89,203 +59,101 @@ You can specify the project files in `polymer.json` so that commands like `polym
...
 
-Installs Bower dependencies, optionally installing multiple variants.
+Run `polymer help serve` for the full list of available options.
```
Comment (rictic) [inline README.md]: I dunno if repeating this is helpful, it's listed at the top.

File: README.md [hunk 11]
```diff
@@ -89,203 +59,101 @@ You can specify the project files in `polymer.json` so that commands like `polym
...
-You can also list specific files to lint and rules to use on the command line:
-
-    $ polymer lint --rules polymer-2-hybrid --input ./my-element.html
-
-### test
-
-Run test with web-component-tester:
-
-    $ polymer test
+Run `polymer help lint` for the full list of available options.
```
Comment (rictic) [inline README.md]: full list of available options and rule sets?

File: README.md [hunk 12]
```diff
@@ -89,203 +59,101 @@ You can specify the project files in `polymer.json` so that commands like `polym
...
+Lint your project for common errors. Provide a set of linting rules via either your `polymer.json` configuration file or the `--rules` command option.
 
-This performs a Bower install, and also installs any dependency variants
-specified in the `"variants"` property of `bower.json`. See
-[Dependency Variants](#dependency-variants) above.
-
-### lint
-
-Add a "lint" section to your `polymer.json` file at the root of your project:
+For example, if you wanted to lint a Polymer 2.0 Hybrid element, you would run `polymer lint --rules polymer-2-hybrid`. To make sure the correct rule set is always used, you should add the following field to your `polymer.json`:
```
Comment (rictic) [inline README.md]: To make sure the correct rule set is always used, you can add a "lint" section to your `polymer.json` like so:

Comment (rictic) [inline README.md]: Do other docs capitalize hybrid?

File: README.md [hunk 13]
```diff
@@ -89,203 +59,101 @@ You can specify the project files in `polymer.json` so that commands like `polym
...
-Use `index.html` as the entrypoint, or read from `polymer.json`:
+Run `polymer help test` for the full list of available options.
 
-    $ polymer build
 
-`build` is opinionated and defaults to a good build for app-shell apps. It writes the built output to `build/bundled` and `build/unbundled` folders. Both outputs have been run though HTML, JS and CSS optimizers, and have a Service Worker generated for them. The bundled folder contains the application files process by Vulcanize, Polymer's HTML bundler, for optimal loading via HTTP/1. The unbundled folder is optimized for HTTP/2 + Push.
+### `polymer build [options...]`
 
-While the build command should support most projects, some users will need greater control over their build pipeline. If that's you, check out the [polymer-build](https://github.com/Polymer/polymer-build) library. Polymer-build can be called and customized programmatically, giving you much greater control than the CLI can provide. Visit the repo for usage information and examples.
+Build a Polymer application for production. This includes support for site optimizations like code bundling, minification, and ES6 compilation to run on older browsers.
```
Comment (rictic) [inline README.md]: site optimizations -> optimizations

File: README.md [hunk 14]
```diff
@@ -89,203 +59,101 @@ You can specify the project files in `polymer.json` so that commands like `polym
...
 
-#### --js-compile
+Most optimizations and production enhancements are disabled by default. To make sure the correct build enhancements are always used, you can provide a set of build configurations via the ["builds"](https://www.polymer-project.org/2.0/docs/tools/polymer-json#builds) field of your `polymer.json` file:
```
Comment (rictic) [inline README.md]: Maybe nix the `production enhancements` part, as it's well covered by `optimizations`

File: README.md (hunk 4 above)
Comment (FredKSchott) [inline README.md]: 👍 going with "during development"

File: README.md (hunk 7 above)
Comment (FredKSchott) [inline README.md]: yea that's a fair point, no need to confuse here.

I'd still like to document that this is supported somewhere but maybe just a reference to `polymer help init` will do.

File: README.md (hunk 8 above)
Comment (FredKSchott) [inline README.md]: Yes, logically it's not required but it helps to point out that there are significant features hidden behind the commands where it's included. For example, I think this lets us get around having to go into detail to explain `--open` and others

File: README.md (hunk 10 above)
Comment (FredKSchott) [inline README.md]: See comment above re: `[...options]`. This lets us point out that there are a collection of important features not listed here which are explained in more detail by `polymer help serve`.

Maybe there's a more clear way to signify this? Any ideas?

File: README.md (hunk 12 above)
Comment (FredKSchott) [inline README.md]: I don't think so, removing...

File: README.md (hunk 1 above)
Comment (FredKSchott) [inline README.md]: +1, also added a note that we explicitly recommend yarn

File: README.md (hunk 9 above)
Comment (FredKSchott) [inline README.md]: mentioned below.

**Comment by FredKSchott** [comment]
@rictic @aomarks @justinfagnani PTAL

File: README.md [hunk 15]
```diff
@@ -23,17 +23,18 @@ The command-line tool for Polymer projects and Web Components.
...
 
 ```bash
-$ npm install -g polymer-cli
-# or...
 $ yarn add global polymer-cli
+# or...
+$ npm install -g polymer-cli
 ```
 
+For best results and a faster installation, we reccomend installing with [yarn](https://yarnpkg.com/en/).
```
Comment (aomarks) [inline README.md]: recommend :)

File: README.md [hunk 16]
```diff
@@ -3,79 +3,50 @@
...
+```bash
+$ yarn add global polymer-cli
+# or...
+$ npm install -g polymer-cli
+```
 
-  * entrypoint - The first file served by the web server for every valid route (usually index.html). This file should be very small, since it may not cache well and must reference resources with absolute URLs, due to being served from many URLs.
-  * shell - The actual app shell, which includes the top-level logic, routing, and so on.
-  * fragments - lazy loaded parts of the application, typically views and other elements loaded on-demand.
+For best results and a faster installation, we reccomend installing with [yarn](https://yarnpkg.com/en/).
```
Comment (abdonrd) [inline README.md]: `reccomend` to `recommend` :)

File: README.md [hunk 17]
```diff
@@ -89,203 +60,101 @@ You can specify the project files in `polymer.json` so that commands like `polym
 }
```
Comment (abdonrd) [inline README.md]: ^^^
`extraDependencies` to `includeDependencies` :)

File: README.md (hunk 15 above)
Comment (FredKSchott) [inline README.md]: doh!

File: README.md (hunk 17 above)
Comment (FredKSchott) [inline README.md]: wow, didn't realize this was THAT old. Good catch.

File: README.md [hunk 18]
```diff
@@ -3,289 +3,158 @@
...
   "fragments": [
-    "src/app-home/app-home.html",
-    "src/app-view-1/app-view-1.html",
+    "src/app-home.html",
+    "src/app-view-1.html",
+    "src/app-view-2.html",
+    "src/app-view-3.html"
   ],
-  "sources": [
+  "sourceGlobs": [
```
Comment (abdonrd) [inline README.md]: Nooo! haha
It is only `sources`. :)

Comment (FredKSchott) [inline README.md]: hahaha I checked this before committing, I promise. Re-pushed :)

File: README.md [hunk 19]
```diff
@@ -3,289 +3,158 @@
...
+```bash
+$ yarn add global polymer-cli
+# or...
+$ npm install -g polymer-cli
+```
 
-  * entrypoint - The first file served by the web server for every valid route (usually index.html). This file should be very small, since it may not cache well and must reference resources with absolute URLs, due to being served from many URLs.
-  * shell - The actual app shell, which includes the top-level logic, routing, and so on.
-  * fragments - lazy loaded parts of the application, typically views and other elements loaded on-demand.
+For best results and a faster installation, we recom,end installing with [yarn](https://yarnpkg.com/en/).
```
Comment (abdonrd) [inline README.md]: Sorry, but... `recommend ` 🙈

Comment (FredKSchott) [inline README.md]: ... jeeze, I need to take a break

Comment (abdonrd) [inline README.md]: You need it! ☕️ 🙂

**Comment by usergenic** [comment]
the CI failures are concerning even if intermittent.  would be nice if we could understand and/or fix reliably before we cut a new release

**Comment by FredKSchott** [comment]
@usergenic I agree. @aomarks is fixing the appveyor issue, but it looks like Travis hit a *separate* yarn issue around scoped packages & possibly travis. See https://github.com/yarnpkg/yarn/issues/1619 & https://github.com/yarnpkg/yarn/issues/2750

I suggest we keep an eye on the yarn+travis issue, if we continue to see it we  may want to investigate more or even switch back to npm.

File: README.md [hunk 20]
```diff
@@ -3,289 +3,158 @@
...
-A command-line tool for Polymer projects.
+The command-line tool for Polymer projects and Web Components.
 
-For detailed help, check out the
-[Polymer CLI](https://www.polymer-project.org/1.0/docs/tools/polymer-cli)
-guide.
+## Features
 
-## Overview
+  - **init** - Create a new Polymer project from pre-configured starter templates
```
Comment (justinfagnani) [inline README.md]: I'd either drop "pre-configured" or change to "built-in or third-party"

File: README.md [hunk 21]
```diff
@@ -3,289 +3,158 @@
...
-App-shell apps are currently the preferred style for Polymer CLI, and most commands are being optimized to support them. App-shell apps usually have client-side routing (see the [app-route](https://github.com/PolymerElements/app-route) element), and lazy load parts of the UI on demand.
-
-Polymer-CLI supports this style by understand these different types of files:
+```bash
+$ yarn add global polymer-cli
+# or...
+$ npm install -g polymer-cli
+```
 
-  * entrypoint - The first file served by the web server for every valid route (usually index.html). This file should be very small, since it may not cache well and must reference resources with absolute URLs, due to being served from many URLs.
```
Comment (justinfagnani) [inline README.md]: I think we need to keep some of this content, since we reference these things in configuration and flags

File: README.md [hunk 22]
```diff
@@ -3,289 +3,158 @@
...
-  * shell - The actual app shell, which includes the top-level logic, routing, and so on.
-  * fragments - lazy loaded parts of the application, typically views and other elements loaded on-demand.
+For best results and a faster installation, we recommend installing with [yarn](https://yarnpkg.com/en/).
 
 ## Configuration
 
-The project files are specified either as global flags: `--entrypoint`, `--shell` and zero or more `--fragment` flags, or in a `polymer.json` configuration file.
+An application can be configured for all commands via global flags: `--entrypoint`, `--shell`, etc.
```
Comment (justinfagnani) [inline README.md]: This sounds pretty vague to me. An application? configured for what? etc? We haven't even mentioned the difference between applications elements by this point

File: README.md [hunk 23]
```diff
@@ -3,289 +3,158 @@
...
-
-* Application projects
-
-  Application projects are self-contained and intended to be deployed as a standalone website. Application projects contain elements in a `src/` folder and import their dependencies with absolute paths, or relative paths that reference folders inside the project folder.
-
-### Application Styles
-
-Polymer-CLI currently supports two styles of applications:
-
-  * Monolithic applications, which have a single entrypoint (usually index.html) and eagerly import all dependencies.
```
Comment (justinfagnani) [inline README.md]: This difference is important and needed to understand other options below.

File: README.md [hunk 24]
```diff
@@ -3,289 +3,158 @@
...
+    "src/app-view-2.html",
+    "src/app-view-3.html"
   ],
   "sources": [
     "src/**/*",
     "images/**/*",
     "bower.json"
   ],
-  "includeDependencies": [
+  "extraDependencies": [
```
Comment (justinfagnani) [inline README.md]: can we drop extraDependencies from the example?

File: README.md [hunk 25]
```diff
@@ -3,289 +3,158 @@
...
-    "src/app-view-1/app-view-1.html",
+    "src/app-home.html",
+    "src/app-view-1.html",
+    "src/app-view-2.html",
+    "src/app-view-3.html"
   ],
   "sources": [
     "src/**/*",
     "images/**/*",
     "bower.json"
```
Comment (justinfagnani) [inline README.md]: can we remove bower.json?

File: README.md [hunk 26]
```diff
@@ -3,289 +3,158 @@
...
+The command-line tool for Polymer projects and Web Components.
 
-For detailed help, check out the
-[Polymer CLI](https://www.polymer-project.org/1.0/docs/tools/polymer-cli)
-guide.
+## Features
 
-## Overview
+  - **init** - Create a new Polymer project from pre-configured starter templates
+  - **install** - Install bower dependencies as well as [dependency variants](https://www.polymer-project.org/2.0/docs/glossary#dependency-variants) for testing
```
Comment (justinfagnani) [inline README.md]: IMO, the glossary link doesn't provide enough detail to replace the variants section here.

File: README.md [hunk 27]
```diff
@@ -3,289 +3,158 @@
...
-    $ polymer init element
 
-You can download and run templates built by our community as well. [Search npm](https://www.npmjs.com/search?q=generator-polymer-init) for the full list.
+### `polymer serve [options...]`
 
-### install
+Start a development server designed for serving Polymer & Web Component projects. Applications are served as-is, while elements are served from a special route where it can properly reference its dependencies.
 
-Installs Bower dependencies, optionally installing multiple variants.
+Additionally, the development server will automatically use [Babel](https://babeljs.io) to transpile any ES6 code down to ES5 for browsers that don't have native support for important ES6 features like classes.
```
Comment (justinfagnani) [inline README.md]: Since this is probably important enough, I'd mention `--compile=always|never|auto`

File: README.md [hunk 28]
```diff
@@ -3,289 +3,158 @@
...
-Start the development server:
+Run `polymer help build` for the full list of available options & optimizations.
 
-    $ polymer serve
+If you need support for something that is missing from the CLI, check out the [polymer-build](https://github.com/Polymer/polymer-build) library. Is the JS library that powers the CLI, and calling it directly gives you much greater control than the CLI can provide. Visit the repo for usage information and examples.
 
-Start the development server, and open the default browser:
 
-    $ polymer serve -o
+### `polymer analyze [options...]`
```
Comment (justinfagnani) [inline README.md]: replace options with `[files...]`

File: README.md [hunk 29]
```diff
@@ -3,289 +3,158 @@
...
 ## Compiling from Source
 
-    $ npm run build
-
 You can compile and run the CLI from source by cloning the repo from Github and then running `npm run build`. But make sure you have already run `npm install` before building.
 
-## Supported node.js versions
+```bash
+# clone the repo from github
+npm install
```
Comment (justinfagnani) [inline README.md]: replace npm with yarn

File: README.md (hunk 20 above)
Comment (FredKSchott) [inline README.md]: I want to be short and simple here so "built-in or third-party" is too wordy. I like "pre-configured" here but if that's a blocker for you I can remove.

File: README.md (hunk 26 above)
Comment (FredKSchott) [inline README.md]: 👍

File: README.md (hunk 23 above)
Comment (FredKSchott) [inline README.md]: This and others are described on polymer-project.org, I don't want to duplicate that here. I'll go through and add some better links to the description there.

File: README.md (hunk 21 above)
Comment (FredKSchott) [inline README.md]: see response above

File: README.md (hunk 22 above)
Comment (FredKSchott) [inline README.md]: 👍 good point, will explain more

File: README.md (hunk 25 above)
Comment (FredKSchott) [inline README.md]: Not until we remove support from polyserve :(
https://github.com/Polymer/polyserve/blob/84752470189f75efa11f2a2dd75572a8b84e292c/src/test/make_app_test.ts#L68

File: README.md [hunk 30]
```diff
@@ -3,289 +3,146 @@
...
-## Configuration
 
-The project files are specified either as global flags: `--entrypoint`, `--shell` and zero or more `--fragment` flags, or in a `polymer.json` configuration file.
+### `polymer install [--variants]`
 
-### polymer.json
+Install your dependencies, similar to running `bower install`.
 
-You can specify the project files in `polymer.json` so that commands like `polymer build` work without flags:
+If the `--variants` option is provided, the command will also search your project's `bower.json` for a `"variants"` property and install any [dependency variants](https://www.polymer-project.org/2.0/docs/glossary#dependency-variants) listed there as well as well. This is useful if you need to test your elements against multiple versions of Polymer and/or sets of dependencies.
```
Comment (FredKSchott) [inline README.md]: lol just noticed `as well as well`

File: README.md (hunk 26 above)
Comment (FredKSchott) [inline README.md]: described in more detail in section below

**Comment by FredKSchott** [comment]
@justinfagnani thanks for the good feedback, PTAL

File: README.md [hunk 31]
```diff
@@ -3,289 +3,146 @@
...
+  - [`shell`](https://www.polymer-project.org/2.0/docs/tools/polymer-json#shell) (Optional): The app shell.
+  - [`fragments`](https://www.polymer-project.org/2.0/docs/tools/polymer-json#fragments) (Optional): A list of other entrypoints into your application.
+  - `root` (Defaults to current working directory): The web root of your application, can be a subfolder of your project directory.
+  - `sources` (Defaults to `src/**/*`): The source files in your application.
 
-  Element projects contain one or more reusable element definitions, intended to be used from other elements, applications or pages. Element definitions live at the top level of the project so they are easy to import. Elements import their dependencies with relative paths that reference sibling folders to the project folder.
+Configuration can be passed for all commands via the global CLI: `--entrypoint`, `--shell`, etc. However we recommend saving your configuration to a `polymer.json` configuration file in your project. This guarantees a single shared configuration that will be read automatically for every command. Other project settings, like build and lint rules, can also be defined here.
```
Comment (justinfagnani) [inline README.md]: "via the global **flags**"

File: README.md [hunk 32]
```diff
@@ -3,289 +3,146 @@
...
-Polymer-CLI currently supports two styles of applications:
+### `polymer help [COMMAND]`
 
-  * Monolithic applications, which have a single entrypoint (usually index.html) and eagerly import all dependencies.
+Run `polymer help` to get a helpful list of supported commands. Pass it a command name (ex: `polymer help serve`) to get detailed information about that command and the options it supports.
 
-  * "App shell" applications, which have a very lightweight entrypoint, an app-shell with startup and routing logic, and possibly lazy loaded fragments.
 
-### App-shell Structure
+### `polymer init`
```
Comment (justinfagnani) [inline README.md]: polymer init **[template]**

Comment (FredKSchott) [inline README.md]: This was already discussed here: https://github.com/Polymer/polymer-cli/pull/675#discussion_r112048237

I'm okay with either, but I'm going to add it back for now to match the rest of the document. Happy to discuss more and remove again if we still feel its not useful here.

**Comment by FredKSchott** [comment]
Phew, thanks everyone!
